js/README.md

```diff
@@ -313,6 +314,9 @@ By default, the main bundle file `ort.min.js` of ONNX Runtime Web contains all f
 |ort.wasm.min.js|148.56|44KB|X|O|O|O|X|
 |ort.wasm-core.min.js|40.56KB|12.74KB|X|O|X|X|X|
 
+#### Build ONNX Runtime as a WebAssembly static library
+
+When `--build_wasm_static_lib` is given instead of `--build_wasm`, it builds a WebAssembly static library of ONNX Runtime and creates a `libonnxruntime_webassembly.a` file at a build output directory. Developers who have their own C/C++ project and build it as WebAssembly with ONNX Runtime, this build option would be useful. This static library is not published by a pipeline, so a manual build is required if necessary.
```

Comment:
ONNX Runtime tries to avoid publishing static libs. ORT has a lot of dependencies like protobuf/onnx. Who wants to use the static lib also need to copy these 3rd-party dependencies header file/libs to their project. And we can't publish them as we can't publish protobuf libs on behalf of Google.  As it is so difficult to use, I believe most people will use it in wrong ways. If it still what you want, you may need to improve the document a bit.

Reply:
I'll add more details into onnxruntime.ai docs.